testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/ReadFeatureDescriptionTestCase.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.as.test.integration.management.api;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.ANNOTATION;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.CHILDREN;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.FEATURE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PACKAGES;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PARAMS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PROVIDES;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.READ_FEATURE_DESCRIPTION_OPERATION;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RECURSIVE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.NAME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.REFS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.REQUIRES;
+
+import java.io.IOException;
+
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.integration.management.base.ContainerResourceMgmtTestBase;
+import org.jboss.as.test.integration.management.util.MgmtOperationException;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.Property;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Test of read-feature-description handling.
+ *
+ * @author Brian Stansberry
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@Ignore
```

Comment:
If you remove this @Ignore we can retest this and merge it after the next core upgrade without having to come back to you first.  The PR is tracked as needing that upgrade so it's ok that the test will fail.